warehouse/redshift/redshift.go

```diff
@@ -50,8 +50,15 @@ var dataTypesMap = map[string]string{
 }
 
 var primaryKeyMap = map[string]string{
-	"users":      "id",
-	"identifies": "id",
+	"users":                      "id",
+	"identifies":                 "id",
+	warehouseutils.DiscardsTable: "row_id",
+}
+
+var partitionKeyMap = map[string]string{
+	"users":                      "id",
+	"identifies":                 "id",
+	warehouseutils.DiscardsTable: "row_id, column_name",
```

Comment:
Since we share a common discards table for all tables, partition keys need to also include table_name, for the rare case that two tables share the same id values with common custom properties names.

The same applies to biquery & snowflake implementations

Reply:
@fxenik added the `table_name`